### ISSUE TITLE
Other training data results

### ISSUE THREAD

**Issue by genbei** [TharinduDR/TransQuest#18]
I ran the enzh example of wmt20 and got pearson =0.6. Then I replaced the enzh data of ccmt19 and replaced the files of train, dev and test-blind. The rest has not changed and got pearson =0.12 .Why, by the way, do you need a vocabulary or do you need to adjust the parameters?

**Comment by leonodelee** [comment]
I can't get your model via your url in https://tharindudr.github.io/TransQuest/models/sentence_level_pretrained/ ; It's said "The connection has been reset"

**Comment by genbei** [comment]
> I can't get your model via your url in https://tharindudr.github.io/TransQuest/models/sentence_level_pretrained/ ; It's said "The connection has been reset"
i run `python -m examples.sentence_level.wmt_2020_task2.en_zh.monotransquest`

**Comment by leonodelee** [comment]
> e

但是这样只能跑例程，怎么应用呢？我跑的2020enzh得到：RMSE=0.9774、MAE=0.7879、Pearson Correlation=0.4865、Spearman Correlation=0.4866，

**Comment by genbei** [comment]
我仅仅替换了数据，可能并不能这样使用吧


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "TharinduDR/TransQuest"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月12日(星期三) 下午2:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [TharinduDR/TransQuest] Other training data results (#18)





  
e
  
但是这样只能跑例程，怎么应用呢？我跑的2020enzh得到：RMSE=0.9774、MAE=0.7879、Pearson Correlation=0.4865、Spearman Correlation=0.4866，

**Comment by TharinduDR** [comment]
TransQuest does not need a vocabulary. In your case I think it might be the case of parameters, Can you try with a different learning rate? 
What is the source in ccmt19?

**Comment by genbei** [comment]
> TransQuest does not need a vocabulary. In your case I think it might be the case of parameters, Can you try with a different learning rate?
> What is the source in ccmt19?
ccmt is the China Conference on Machine Translation
Excuse me, where can I adjust the parameters? I am not familiar with the code yet

**Comment by TharinduDR** [comment]
There is a monotransquest_config.py in the same folder, you can adjust the learning rate there. 
By source, what I meant was, source of the texts. The training data for this model was taken from Wikipedia. Therefore XLM-R model performed well. If your data is from a different source like Twitter, XLM-R model won't perform well.

**Comment by genbei** [comment]
> There is a monotransquest_config.py in the same folder, you can adjust the learning rate there.
> By source, what I meant was, source of the texts. The training data for this model was taken from Wikipedia. Therefore XLM-R model performed well. If your data is from a different source like Twitter, XLM-R model won't perform well.

What should I do if I want to train myself? I didn't see the relevant introduction.In addition, did you train XLM-R yourself?

**Comment by TharinduDR** [comment]
Please have a look at the published paper, I used the XLM-R large model
